boards/shields/mikroe_eth_click/mikroe_eth_click.conf

```diff
@@ -0,0 +1,7 @@
+# ENC28J60 is L2 chip slave on SPI
+CONFIG_SPI=y
+CONFIG_NET_L2_ETHERNET=y
+
+# ENC28J60 Ethernet Device
+CONFIG_ETH_ENC28J60=y
+CONFIG_ETH_ENC28J60_0=y
```

Comment:
@galak is this instance config going away too?

Reply:
at some point in the future, this is fine for now.